lib/dune

```diff
@@ -0,0 +1,19 @@
+(library
+  (name duniverse_lib)
+  (libraries
+    bos
+    ezjsonm
+    fmt
+    logs
+    opam-core
+    opam-file-format
+    rresult
+    sexplib
+    uri
+  )
+  (preprocess
+    (per_module
```

Comment:
this was in the original dune file, but I'd say not to bother with this optimization, this can be confusing when adding a converter in another place